modules/benchmark/R/create.BRR.R

```diff
@@ -0,0 +1,54 @@
+##-------------------------------------------------------------------------------------------------#
+##' For each benchmark id, calculate metrics and update benchmarks_ensemble_scores
+##'  
+##' @name create.BRR
+##' @title Create benchmark reference run and ensemble
+##' @param ensemble.id id of ensemble run that will be used for benchmarking
+##' @param con database connection
+##' @export 
+##' 
+##' @author Betsy Cowdery 
+
+create.BRR <- function(ensemble.id, workflow, con){
+
+cnd1 <- workflow$hostname == fqdn() 
+cnd2 <- workflow$hostname == 'test-pecan.bu.edu' & fqdn() == 'pecan2.bu.edu'
+cnd3 <- workflow$hostname == 'pecan2.bu.edu' & fqdn() == 'test-pecan.bu.edu'
+  
+  
+if(cnd1|cnd2|cnd3){  # If the ensemble run was done on localhost, turn into a BRR
+  
+    BRR <- db.query(paste0("INSERT INTO reference_runs (model_id, settings, user_id, created_at, updated_at) VALUES(",workflow$model_id,", '",workflow$folder,"' , ",user_id,", NOW() , NOW()) RETURNING *;"),con)
+    
+    bm.ensemble <- db.query(paste0("INSERT INTO benchmarks_ensembles (reference_run_id, ensemble_id, model_id, user_id, created_at, updated_at, citation_id) VALUES(",BRR$id,",",ensemble.id,",", BRR$model_id,", ",user_id,", NOW() , NOW(), 1000000001 ) RETURNING *;"),con)
+    
+    warning("Because you are creating a new reference run you will need to manually relate benchmarks.  This is done by 1) creating records for new data sources using the formats, inputs and dbfiles tables 2) Relating benchmarks with the reference run in the benchmarks_benchmarks_reference_runs table.")
```

Comment:
should use the logger functions, not warn. Also, if this function isn't creating benchmarks, we need to work through what function IS doing that because a text warning is insufficient.